src/Rx.ts

```diff
@@ -184,6 +184,8 @@ import observable from 'symbol-observable';
  * asynchronous conversions.
  * @property {Scheduler} async Schedules work with `setInterval`. Use this for
  * time-based operations.
+ * @property {Scheduler} animation Schedules work with `requestAnimationFrame`.
```

Comment:
@blesh ok, I'll update. Though I was thinking `Scheduler.animation` vs. `Scheduler.animationFrame` would be a nice alias.